feat: external tools if not installed run via docker (#314)
crashappsec/chalk: CHANGELOG.md

  - New key holding GCP project metadata: `_GCP_PROJECT_METADATA`
    ([#311](https://github.com/crashappsec/chalk/pull/31))

> - Chalk external tools (syft and semgrep) when not already
>   installed, if docker is available, run via docker vs
>   installing them on the host system.
>   This avoid requiring any system requirements to install
>   the external tool.
>   ([#314](https://github.com/crashappsec/chalk/pull/314))

Author: python3 and pipx are not required by all plugins. the wording explicitly was generic that each tool can have its own system requirements which are avoided due to docker

```suggestion
- The Chalk external tools `syft` and `semgrep` are now run via Docker
  when they are not installed and Docker is available. This avoids the
  need to install them on the host system.
  ([#314](https://github.com/crashappsec/chalk/pull/314))
```
